valpas-web/test/integrationtests/kuntarouhinta.test.ts

```diff
@@ -0,0 +1,125 @@
+import { kuntarouhintaPathWithOid } from "../../src/state/paths"
+import {
+  clickElement,
+  expectElementEventuallyVisible,
+  textEventuallyEquals,
+} from "../integrationtests-env/browser/content"
+import { urlIsEventually } from "../integrationtests-env/browser/core"
+import {
+  dataTableEventuallyEquals,
+  getExpectedRowCount,
+} from "../integrationtests-env/browser/datatable"
+import { loginAs } from "../integrationtests-env/browser/reset"
+import { helsinginKaupunkiOid } from "./oids"
+import { selectOrganisaatioByNimi } from "./organisaatiovalitsin-helpers"
+
+const helsinginKuntarouhintaPath = kuntarouhintaPathWithOid.href(
+  "/virkailija",
+  { organisaatioOid: helsinginKaupunkiOid }
+)
+
+const helsinginKuntarouhintaTableContents = `
```

Comment:
Olisi varmaan hyvä, että olisi joku, kenellä on myös keskeytyksiä. Laitoin lapun taululle.

Reply:
Eilen lisäämäni Pyhtää olisi myös varmaan parempi ja stabiilimpi testitapaus, pyhtäälle merkittyjä oppijoita on vain kourallinen eikä näin pitkää listaa. Laputin tämänkin.